e2e-tests/cypress/support/appeal-statement-submission/confirmAppealStatementFileIsUploaded.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (filename) => {
   cy.goToAppealStatementSubmission();
-  cy.get('#appeal-upload-file-name')
+  cy.get('[data-cy="appeal-upload-file-link"]')
```

Comment:
all follow the same pattern - swapped from a `span` to an `a` tag nested in the `span`, so now getting the text from the anchor.